shenyu-admin/src/main/java/org/apache/shenyu/admin/service/configs/ConfigsExportImportEnum.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.admin.service.configs;
+
+public enum ConfigsExportImportEnum {
+
+    /**
+     * auth.
+     */
+    Auth("auth.json", 0),
+
+    /**
+     * meta.
+     */
+    Meta("meta.json", 1),
+
+    /**
+     * plugin.
+     */
+    Plugin("plugin.json", 2),
+
+    /**
+     * selector.
+     */
+    Selector("selector.json", 3),
+
+    /**
+     * rule.
+     */
+    Rule("rule.json", 4),
+
+    /**
+     * dict.
+     */
+    Dict("dict.json", 5),
```

Comment:
may include dict and plugin handler?

Reply:
Currently, it includes both of these

Reply:
I think you should add plugin handler

Reply:
The plugin handler is associated with the plugin